src/netrc.js

```diff
@@ -0,0 +1,32 @@
+const core = require('@actions/core');
+const fs = require('fs');
+const path = require('path');
+
+module.exports.getNetrcPath = function () {
```

Comment:
For those who aren't too familiar with netrc, could you explain the usage via comments here?

Reply:
OMG, you just made me think of a way we can get rid of this entirely! New commit coming soon…

Reply:
OK, with https://github.com/readmeio/heroku-review-app-action/pull/1/commits/191e9ce4f42f1b06ee3cf3be79808d5a2ba29f63 we're no longer using a `.netrc` file for the Heroku credentials; instead we're putting it in the username and password fields of the HTTP URL.

Example: the URL was previously `https://git.heroku.com/metrics-test-server-pr-1234.git`, now it's `https://ryanp%40readme.io:abcdef@git.heroku.com/metrics-test-server-pr-1234.git`.

This is much less code, and also much cleaner because we don't have to leave our Heroku credentials sitting around in a file on the filesystem. (I'm confident GitHub Actions would clean up that file, but also, it felt a bit sketchy to do that in an environment we don't directly control…)